libs/atqr/domain/src/lib/types/notification.types.ts

```diff
@@ -0,0 +1,32 @@
+import { ITemplateDataTypes } from '../services/notification/templateDataType.interface';
+
+export type TemplateDataTypes = {
+  [K in keyof ITemplateDataTypes]: ITemplateDataTypes[K];
+};
+
+export enum MessageTemplateId {
+  ChallengeCreated = 'ChallengeCreated',
+  ChallengeCanceled = 'ChallengeCanceled',
+  ChallengeOngoing = 'ChallengeOngoing',
+  ChallengeCompleted = 'ChallengeCompleted',
+  ChallengeFailed = 'ChallengeFailed',
+  YourOfficiationHaveBeenRequested = 'YourOfficiationHaveBeenRequested',
+  YourOfficiationRequestHaveBeenAccepted = 'YourOfficiationRequestHaveBeenAccepted',
+  YourOfficiationRequestHaveBeenRejected = 'YourOfficiationRequestHaveBeenRejected',
+  YourOfficiationRequestHaveBeenIgnored = 'YourOfficiationRequestHaveBeenIgnored',
+  YouHaveBeenChallenged = 'YouHaveBeenChallenged',
+  YourContenderAcceptedTheChallenge = 'YourContenderAcceptedTheChallenge',
+  YourContenderRejectedTheChallenge = 'YourContenderRejectedTheChallenge',
+  YourContenderIgnored = 'YourContenderIgnored',
+  YourContenderRemovedYouFromJudgeRole = 'YourContenderRemovedYouFromJudgeRole',
+  YourContenderCanceledTheChallenge = 'YourContenderCanceledTheChallenge',
+  YourContenderWereTheySuccessful = 'YourContenderWereTheySuccessful',
```

Comment:
"Your Contender Were They Successful" is grammatically incorrect. Did you mean "Your contender succeeded"? (again, check usage and also fix Disl)

Reply:
Originally it is WereTheySuccessful , as a message template askingif the contender was successful . Don'1 like the name very much kk if you`d like I can make it more semantic

Reply:
Since all other contender notifications are written as if there was only one contender, let's keep it that way for this too. But my point was that the expression "were they" is wrong. It should be "Your contenderS were successful", "your contendeR was successful", or, if it was a question, "Were (they or your contenderS) successful"

So keep it "YourContenderWasSuccessful"

Reply:
yes, it was not me who wrote this . I already corrected it